### PR TITLE
Gashapon, again!

### PR DESCRIPTION
**Pull request by NigglyWiggly**
More Gashapon toys, because why not?

### CODE REVIEW

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -823,6 +853,48 @@
...
+/obj/item/toy/gasha/mario
+	name = "toy plumber"
+	desc = "It's a toy of a popular plumber character.
+	icon_state = "mario"
+
+/obj/item/toy/gasha/mario/luigi
+	icon_state = "luigi"
+
+/obj/item/toy/gasha/mario/star
+	icon_state = "star"
```
Comment (Intigracy) [inline code/game/objects/items/toys.dm]: This one is inheriting mario's desc, and it's not really a plumber character, is it?

File: code/game/objects/items/toys.dm [hunk 2]
```diff
@@ -823,6 +853,48 @@
...
+	name = "Robot"
+	desc = "It's a small robot toy"
+	icon_state = "quote"
+
+/obj/item/toy/gasha/quote/curly
+	icon_state = "curly"
+
+/obj/item/toy/gasha/mario
+	name = "toy plumber"
+	desc = "It's a toy of a popular plumber character.
```
Comment (Intigracy) [inline code/game/objects/items/toys.dm]: Missing a " here

File: code/game/objects/items/toys.dm (hunk 1 above)
Comment (NigglyWiggly) [inline code/game/objects/items/toys.dm]: It is, it's mario flashing like when you grab a star

**Comment by Probe1** [comment]
Adding an active players name is too much in my opinion and would be a precedent in circlejerk levels.  I do not like it and the pussies in the thread won't attach a name to it but they don't like it either.  

I do support things like rambling maniac because he is no longer active and had a lasting effect on the memory of those who played when he did.  Most importantly, he does not play anymore.  A tribute to a former player is nice but to an active player is metaclubby.

**Comment by Boxe** [comment]
i vote rename the defsquid thing to deathsquad so admins have an easier time spawning it in the rare event that they may have to

**Comment by Ephx1** [comment]
Where are my BORERS?

**Comment by NigglyWiggly** [comment]
The only one of the horrible namefag circlejerk ones that actually spawns in regular play is the Rambling Maniac one.
Mostly this is just to fuck with Nevill.

**Comment by Probe1** [comment]
That's fine with me if it isn't going to actively be in the game.

**Comment by Ephx1** [comment]
Well then, I want a Chaco with accompanying lipstick with twisting action!

**Comment by clusterfack** [comment]
Yeah for future reference I also do not agree with adding references to current players/arguments/etc into the code. Its immulsion shattering seeing IC converted to OOC and shoved back into IC again. (Like when velard put in that anti mommi shitpost into self-emag text)

**Comment by Nienhaus** [comment]
Best toy.
z8fsgwe

**Comment by NigglyWiggly** [comment]
goddammit quit making such cute sprites you bastard

**Comment by Nienhaus** [comment]
You just happen to put up PRs after I made something related :D
But why is it always your PRs.

**Comment by Probe1** [comment]
ED203 IS THE CUTEST!!

**Comment by Nienhaus** [comment]
@NigglyWiggly Add me http://steamcommunity.com/id/Nienhaus

**Comment by ComicIronic** [comment]
REMOVE NAMEFAGS

**Comment by Angelite191** [comment]
I'm going to go ahead and agree with Probe1 and Clusterfack. I honestly can't phrase it better than Probe did.

**Comment by NigglyWiggly** [comment]
Reminder that none of the staticname ones, sans Rambling, appear at all outside of adminbus and the Nevill one is purely so I can use it to fuck with him like the awful person I am.

**Comment by Ephx1** [comment]
I demand it!
chaco

**Comment by NigglyWiggly** [comment]
But if i add mimiga toys people will claim it's furry shit!

Oh god I should add a mimiga mask

**Comment by Ephx1** [comment]
I might as well make the entire furry cast then if it makes people that mad.

**Comment by NigglyWiggly** [comment]
Fine jeez, lemme throw together some mimiga toys then

**Comment by NigglyWiggly** [comment]
Okay there.

**Comment by Ephx1** [comment]
Perfect.

"You get the feeling this one is worst girl". Fite me.

**Comment by NigglyWiggly** [comment]
Chaco a sloot
Sue a qt

File: code/game/objects/items/toys.dm [hunk 3]
```diff
@@ -798,6 +798,36 @@
...
+	name = "toy death squaddie"
+	desc = "Wait what aren't these guys supposed to be top secret or something?"
+	icon_state = "defsquid"
+
+/obj/item/toy/gasha/wizard
+	name = "toy wizard"
+	desc = "This toy is not actually magical."
+	icon_state = "wiz"
+
+/obj/item/toy/gasha/bamshoot
```
Comment (d3athrow) [inline code/game/objects/items/toys.dm]: remove meatclub

File: code/game/objects/items/toys.dm [hunk 4]
```diff
@@ -823,6 +853,73 @@
 	desc = "Harvesters tend to have a bad habit of violently stabbing anyone they meet"
 	icon_state = "harvester"
 
+/obj/item/toy/gasha/narnar
+	name = "toy Nar-Sie"
+	desc = "The father figure to all of his faithful, the Geometer of Blood himself; NAR-SIE!"
```
Comment (d3athrow) [inline code/game/objects/items/toys.dm]: narsie is a lady, in olden times she ate the universe with her gaping vagina of fury

File: code/game/objects/items/toys.dm [hunk 5]
```diff
@@ -823,6 +853,73 @@
...
+	icon_state = "sue"
+
+/obj/item/toy/gasha/mimiga/toroko
+	icon_state = "toroko"
+
+/obj/item/toy/gasha/mimiga/king
+	icon_state = "king"
+
+/obj/item/toy/gasha/mimiga/chaco
+	desc = "It looks like some sort of rabbit-thing, for some reason you get the feeling that this one is the 'worst girl.'"
```
Comment (d3athrow) [inline code/game/objects/items/toys.dm]: >hating chaco

Because you couldn't get the lipstick right

File: code/game/objects/items/toys.dm (hunk 4 above)
Comment (NigglyWiggly) [inline code/game/objects/items/toys.dm]: Nar-sie is a goofy dad, that's why everyone calls him Papa Nar-Sie, jeez

File: code/game/objects/items/toys.dm (hunk 3 above)
Comment (NigglyWiggly) [inline code/game/objects/items/toys.dm]: B-but it's adminbus only!

Comment (Boxe) [inline code/game/objects/items/toys.dm]: the people have spoken

Comment (NigglyWiggly) [inline code/game/objects/items/toys.dm]: Fine Boxxy, I'll make an 'Isaac Harrow' figure too

Comment (Boxe) [inline code/game/objects/items/toys.dm]: no

**Comment by clusterfack** [comment]
Not really controversial anymore, but I'd prefer if you squash commits that do not compile

**Comment by NigglyWiggly** [comment]
What do you mean?

**Comment by clusterfack** [comment]
Squashing commits is a way of removing commits from the history of your changes. You use git rebase then just replace 'pick' with 'squash' for all commits you want to hide.

File: code/game/objects/items/toys.dm [hunk 6]
```diff
@@ -798,6 +798,38 @@
...
+	name = "toy wizard"
+	desc = "This toy is not actually magical."
+	icon_state = "wiz"
+
+/*
+/obj/item/toy/gasha/bamshoot
+	name = "toy Bumshooter"
+	desc = "*fart"
+	icon_state = "bamshoot"
+*/ //No metaclub allowed ;_;
```
Comment (SkowronX) [inline code/game/objects/items/toys.dm]: Remove

File: code/game/objects/items/toys.dm [hunk 7]
```diff
@@ -798,6 +798,38 @@
 	desc = "WE GOT THE VALIDS AI CALL THE SHUTTLE"
 	icon_state = "comdom"
 
+/obj/item/toy/gasha/maniac
+	name = "toy maniac"
+	desc = "NOW WITH REAL KUNG-FU SEIZURE ACTION!"
+	icon_state = "maniac"
```
Comment (SkowronX) [inline code/game/objects/items/toys.dm]: Remove

File: code/game/objects/items/toys.dm (hunk 6 above)
Comment (NigglyWiggly) [inline code/game/objects/items/toys.dm]: It is removed you huge turbonerd

File: code/game/objects/items/toys.dm (hunk 7 above)
Comment (NigglyWiggly) [inline code/game/objects/items/toys.dm]: n o p e

Comment (SkowronX) [inline code/game/objects/items/toys.dm]: No metaclub

Comment (NigglyWiggly) [inline code/game/objects/items/toys.dm]: This was added before he came back, it's not meant to be a metaclub thing you dork

Comment (SkowronX) [inline code/game/objects/items/toys.dm]: Well congratulations, he's an active player now. Time to remove.

**Pull request merged**